src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import type { ChangeEventHandler, HTMLProps, ReactNode } from "react";
+
+import Field from "../Field";
+
+export const FILLED_COLOR = "#0066CC";
+
+type Props = {
```

Comment:
I think we probably still want to have prop-types as well for those not using typescript, as well as giving as much help as possible to people consuming the library.

Reply:
> I think we probably still want to have prop-types as well for those not using typescript, as well as giving as much help as possible to people consuming the library.

Once we've moved off CRA, we should look at generating proptype from types using [babel-plugin-typescript-to-proptypes](https://github.com/milesj/babel-plugin-typescript-to-proptypes).

Reply:
:+1: